metaflow/plugins/kfp/kfp.py

```diff
@@ -575,7 +579,7 @@ def _set_container_resources(
                     operator="Equal",
                     value=accelerator_type,
                 )
-                container_op.add_affinity(affinity)
+                # container_op.add_affinity(affinity)
```

Comment:
consider removing the line completely if not needed.

Reply:
done!